gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -271,7 +271,7 @@ message FailJobRequest {
   // this is particularly useful if a job runs out of retries and an incident is raised,
   // as it this message can help explain why an incident was raised
   string errorMessage = 3;
-  // the backoff timeout for the next retry
+  // the backoff timeout in ms for the next retry
```

Comment:
:wrench: We could use the same style as in other places for the time unit (i.e. `(in ms)`). 

```suggestion
  // the backoff timeout (in ms) for the next retry
```